pkgs/development/interpreters/wamr/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wamr";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "bytecodealliance";
+    repo = "wasm-micro-runtime";
+    rev = "WAMR-${version}";
+    hash = "sha256-jpT42up9HAVJpo03cFrffQQk2JiHEAEepBGlU4RUfNU=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  sourceRoot = "source/product-mini/platforms/linux";
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp iwasm $out/bin
+    runHook postInstall
+  '';
```

Comment:
Why do you write the `installPhase` manually anyway? It seems the default install script just works, and it also installs the library.

```
./result/
├── bin/
│  └── iwasm*
└── lib/
   └── libiwasm.so*
```

It's still kind of strange that `include` is not installed by default though.

```suggestion
```

Reply:
Thanks!